rancher-config/docker-compose.yml

```diff
@@ -30,6 +32,9 @@ services:
       NODE_ENV: production
       JSON_FILE_URL: ${JSON_FILE_URL_PHARMACIES}
       ES_HOST: ${ES_HOST_PHARMACIES}
+      ES_REPLICAS: ${ES_REPLICAS_PHARMACIES}
+      ES_SHARDS: ${ES_SHARDS_PHARMACIES}
+      ES_HOST: ${ES_HOST_PHARMACIES}
```

Comment:
Copy and paste error (it is already specified https://github.com/nhsuk/elasticsearch-updater/pull/29/files#diff-741d52b933c193ee9ab902088ea15391R34)